### PR TITLE
Jormungandr: fix next link for journey pagination

### PR DESCRIPTION
**Pull request by l-vincent-l**
There was a mess between arrival and departure.

Plus we know check only journeys with public transport.
And we were asking for the one leaving one minute after, now we take the one leaving one second after.

### CODE REVIEW

File: source/jormungandr/jormungandr/interfaces/v1/Journeys.py [hunk 1]
```diff
@@ -321,23 +321,28 @@ def extremes(self, resp):
         datetime_before = None
         datetime_after = None
         try:
+            section_is_pt = lambda section : section['type'] == "public_transport"\
+                               or section['type'] == "on_demand_transport"
+            or_func = lambda a,b : a or b
+            journey_as_pt = lambda journey : reduce(or_func, journey['sections'])
```
Comment (antoine-de) [inline source/jormungandr/jormungandr/interfaces/v1/Journeys.py]: ca peut etre sorti du try non ?

Comment (antoine-de) [inline source/jormungandr/jormungandr/interfaces/v1/Journeys.py]: je suis pas certain de bien comprendre, mais en fait tu veux pas faire un any(section_is_pt(s) for s in journey['sections']) ?

**Comment by l-vincent-l** [comment]
test this please

**Comment by l-vincent-l** [comment]
test this please

**Pull request merged**